tools/config/__init__.py

```diff
@@ -795,14 +808,11 @@ def _process_config_and_overrides(self, data, params, unit_name, unit_kind):
                         setattr(self.target, attribute, val)
                         continue
                     else:
+                        if "lib2.test" == name:
+                            print("HERE {}".format(unit_name))
```

Comment:
Is this suppose to be in here?

Reply:
IT's supposed to be "THERE", actually :D